tests/conftest.py

```diff
@@ -39,6 +63,25 @@ def docker(event_loop):
 
 
 @pytest.fixture
+def swarm(event_loop, docker):
```

Comment:
Yes. I'm personally in favor to keep the environment clean if we can or prevent the test to run if we can't. I just assume that both cases exist: the dev can run the tests on their local Docker daemon or they can run it on a dind if they want to test a specific version.